package.json

```diff
@@ -44,7 +44,9 @@
   "release": {
     "repositoryUrl": "https://github.com/VATSIM-UK/uk-controller-plugin",
     "branches": [
-      "main"
+      "main",
```

Comment:
Might just be GH and its syntax highlighting, but the `true` values here are showing as  invalid.